From 58287a65a9d6ca869918566880a39c1fd4a87830 Mon Sep 17 00:00:00 2001
From: Questioner <d00-explanatory-example-debate_c@fair-debate-users.org>
Date: Thu, 27 Aug 2026 08:35:00 +0200
Subject: [PATCH 10/12] add contribution c/a32b38c.md

---
 c/a32b38c.md | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 c/a32b38c.md

diff --git a/c/a32b38c.md b/c/a32b38c.md
new file mode 100644
index 0000000..40299e3
--- /dev/null
+++ b/c/a32b38c.md
@@ -0,0 +1,4 @@
+---
+splitter_version: 2
+---
+::a32b38c1 What are the weak points of this approach?
\ No newline at end of file
-- 
2.39.5

